### PR TITLE
Burger menu

### PR DESCRIPTION
**Pull request by Cleop**
- Creates burger menu to display on mobile (js free), #249 
- Ensures the navbar is fixed at the top, #260 
- Two small refactors: #265, #266

### CODE REVIEW

File: redesign/css/dwyl.css [hunk 1]
```diff
@@ -60,14 +60,22 @@ body {
...
 .mw-21 {
   max-width: 21rem;
 }
 
+/*Navbar*/
+
+.menu {
+  min-height: 4rem;
+  max-height: 0;
+  transition: max-height 1s;
```
Comment (SimonLab) [inline redesign/css/dwyl.css]: The transition feels a bit slow, could we make it a bit faster?

**Comment by Cleop** [comment]
@SimonLab, thanks for your feedback. I'll create issues for these where they don't already exist.
Happy to change 1 and I agree with point 2 definitely (I'll ask @harrygfox for alternative colors).
For points 3 I will look into this and see if I can do it without JS using custom checkboxes. If so, which of the two 'X' designs do you prefer that I mocked in: https://github.com/dwyl/dwyl-site/issues/249#issuecomment-307844707 ?

**Comment by nelsonic** [comment]
Agree with Simon's observation/comment that the background color for the Menu 
could be a different color from the site background to increase contrast. 
(_screenshots for convenience_)

New Homepage on Mobile:
image
Burger Menu Expanded:
image

image

image

If we use `bg-near-black` for the menu the contrast between the menu and content increases:
image

> We need feedback from @iteles  on this.

Additionally I think the dwyl "heart-tick" logo size should be increased.
Otherwise think it looks & works great! 🎉

File: redesign/css/dwyl.css (hunk 1 above)
Comment (nelsonic) [reply redesign/css/dwyl.css]:
```suggestion
transition: max-height 0.5s;
```
😉

Comment (nelsonic) [reply redesign/css/dwyl.css]: Made it faster for you @SimonLab https://github.com/dwyl/learn-tachyons/pull/52/files#diff-910fef80905cec0740d8c9b257c89defR47 😉

**Pull request merged**